e2e_http1_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"crypto/tls"
 	"fmt"
+	"github.com/monzo/terrors"
```

Comment:
nit: can we make sure the imports are appropriately grouped

Reply:
Urgh I think my Goland imports setup for this repo is buggered, let me take a look